### ISSUE TITLE
Add Attachments to topic, not page

### ISSUE THREAD

**Issue by anderson13g** [enterprisemediawiki/SemanticMeetingMinutes#24]
I'm getting a lot of feedback that people are looking for attachments (such as a powerpoint presentation)  in the topic header. They're up in the infobox, but this is no intuitive. If there is more than one topic with more than one attachment, the user also has to select the correct attachment from up above.

**Comment by jamesmontalvo3** [comment]
Good idea. The technical limitation here is that topics are already a multiple-instance template (e.g. you hit the "add topic" button to create another instance), and adding multiple files to a topic would require a multiple-instance template inside the topic. I don't believe it's possible to nest them. Adding a field for a single upload (or two, or any fixed number) could work, but not sure if that's the cleanest solution.
